core/src/main/kotlin/org/evomaster/core/search/FitnessValue.kt

```diff
@@ -146,6 +147,29 @@ class FitnessValue(
                 .count()
     }
 
+    /**
+     * this method is to report the union results with targets at boot-time
+     *
+     * @return first is boot-time (negative means that the boot-time info is unavailable)
+     *         second is search time, and
+     *         third is a union of boot-time and search time
+     */
+    fun unionWithBootTimeCoveredTargets(prefix: String?, idMapper: IdMapper, bootTimeInfoDto: BootTimeInfoDto?, unavailableBootTime: Int = -1): Triple<Int, Int, Int>{
```

Comment:
the use of `Triple` is fine in a local scope, but not when returned by a public method, as it makes code harder to read. eg., how to understand what `first` and `third` are referring to semantically.
create a new class with meaningful names to replace this `Triple`